tests/pytest/eligibility/test_views.py

```diff
@@ -103,6 +104,7 @@ def test_index_get_agency_single_flow(mocker, model_TransitAgency, model_Enrollm
     mock_manager = mocker.Mock()
     mock_manager.all.return_value = [model_EnrollmentFlow]
     type(mock_agency).enrollment_flows = mocker.PropertyMock(return_value=mock_manager)
```

Comment:
I'm curious if these lines are still needed or if they can be removed.

Reply:
Hmm probably not, since we aren't calling the `.all` helper anymore, right? I guess we just need to mock the `.filter` helper.

Reply:
Yeah! It _shouldn't_ need it but for some reason right now it does for tests to pass.

Marking this PR as In Progress again b/c I'm trying to write a different spec to test the filtering function.